scapy/contrib/macsec.uts

```diff
@@ -0,0 +1,267 @@
+# MACsec unit tests
+# run with:
+#   test/run_tests  -P "load_contrib('macsec')" -t scapy/contrib/macsec.uts -F
+
++ MACsec
+~ crypto not_pypy
```

Comment:
Any idea why this does not work with Pypy?

Reply:
My comment is misleading. The PR works fine with pypy on my Debian.

This is related to the pypy version used in Travis, see https://github.com/secdev/scapy/blob/master/.travis/install.sh#L12